applications/nrf5340_audio/unicast_client/main.c

```diff
@@ -255,6 +255,17 @@ static void le_audio_msg_sub_thread(void)
 			ERR_CHK(ret);
 			break;
 
+		case LE_AUDIO_EVT_NO_VALID_CFG:
+			LOG_WRN("No valid configurations found or CIS establishment failed, will "
+				"disconnect");
+
+			ret = bt_mgmt_conn_disconnect(msg.conn, BT_HCI_ERR_REMOTE_USER_TERM_CONN);
```

Comment:
This is very reasonable. Needs a bit of manual testing to verify that all is OK

Reply:
Yes, I did some tests for using PCFT to trigger the CIS created failed, and we can trigger disconnection and then start everything from scratch.